core/block.js

```diff
@@ -38,7 +38,7 @@ const Tooltip = goog.require('Blockly.Tooltip');
 const VariableModel = goog.requireType('Blockly.VariableModel');
 /* eslint-disable-next-line no-unused-vars */
 const Workspace = goog.requireType('Blockly.Workspace');
-const connectionTypes = goog.require('Blockly.connectionTypes');
+const connectionTypes = goog.require('
```

Comment:
1. Something seems to have gone wrong here.  Failed search-and-replace?
2. The local variable should be renamed too—if the module is `Blockly.ConnectionType` then the local alias should be `ConnectionType`.

Here in several other files.